ui/src/components/NewExperimentNext/data/types.ts

```diff
@@ -144,6 +144,106 @@ const ioCommon: Spec = {
   },
 }
 
+const httpCommonSelect: Spec = {
+  port: {
+    field: 'number',
+    label: 'Port',
+    value: 80,
+    helperText: 'The target port of request',
+  },
+  path: {
+    field: 'text',
+    label: 'Path',
+    value: '*',
+    helperText: 'The target URI path of request, support wildcards(https://www.wikiwand.com/en/Matching_wildcards)',
+  },
+  method: {
+    field: 'select',
+    items: ['', 'GET', 'POST', 'PUT', 'DELETE', 'HEAD', 'OPTIONS', 'PATCH'],
```

Comment:
Default `All`?

Then post-processed before submission, `delete` `method` field if the value is `ALL`.

Reply:
Sounds reasonable.

Reply:
👌🏽